values.yaml

```diff
@@ -1040,7 +1040,7 @@ ui:
     provider: "prometheus"
 
     # baseURL is the URL of the prometheus server, usually the service URL.
-    # This value is only used if ui.enabled is set to true.
+    # This value is only used if `ui.enabled` is set to true.
     # @type: string
     baseURL: http://prometheus-server
```

Comment:
Also L1304 needs prometheus.io/path in backticks

Reply:
good catch!!